mythril/analysis/solver.py

```diff
@@ -109,29 +116,37 @@ def get_transaction_sequence(global_state, constraints):
             minimize.append(transaction.call_data.calldatasize)
             minimize.append(transaction.call_value)
 
-            concrete_transactions[tx_id] = tx_template.copy()
-
         else:
             creation_tx_ids.append(tx_id)
 
     model = get_model(tx_constraints, minimize=minimize)
-
+    min_price_dict = {}  # type: Dict[str, int]
     for transaction in transactions:
         tx_id = str(transaction.id)
-
-        concrete_transactions[tx_id]["calldata"] = "0x" + "".join(
+        concrete_transaction = tx_template.copy()
+        concrete_transaction["input"] = "0x" + "".join(
             [
                 hex(b)[2:] if len(hex(b)) % 2 == 0 else "0" + hex(b)[2:]
                 for b in transaction.call_data.concrete(model)
             ]
         )
-
-        concrete_transactions[tx_id]["call_value"] = (
-            "0x%x"
-            % model.eval(transaction.call_value.raw, model_completion=True).as_long()
-        )
-        concrete_transactions[tx_id]["caller"] = "0x" + (
+        value = model.eval(transaction.call_value.raw, model_completion=True).as_long()
+        concrete_transaction["value"] = "0x%x" % value
+        origin = "0x" + (
             "%x" % model.eval(transaction.caller.raw, model_completion=True).as_long()
         ).zfill(40)
+        concrete_transaction["origin"] = origin
+        concrete_transaction["address"] = "%s" % transaction.callee_account.address
+        concrete_transactions.append(concrete_transaction)
+        min_price_dict[origin] = min_price_dict.get(origin, 0) + value
```

Comment:
I don't think origin is necessarily the one who pays the price right? The difference between origin and sender

Reply:
It's acutally caller which is being used, according to mythril origin can literally be anything.Will rename them.